ui/public/scripts/config.local.js

```diff
@@ -1,2 +1,2 @@
 // Use locally running MCP
-window.MCP_URL = 'https://127.0.0.1:3001';
+window.LOCAL_MCP_URL = 'https://127.0.0.1:3001';
```

Comment:
@philbrookes Maybe the name 'GLOBAL_MCP_URL' would make sense now. It would be a feature of sorts then for using a global mcp if one is configured, which works for local dev too.